src/main/java/com/minecolonies/coremod/commands/colonycommands/ShowColonyInfoCommand.java

```diff
@@ -62,14 +63,22 @@ public void execute(@NotNull final MinecraftServer server, @NotNull final IComma
 
         if(colonyId == -1 && args.length >= 1)
         {
-            final EntityPlayer player = server.getEntityWorld().getPlayerEntityByName(args[0]);
+            final GameProfile playerProfile =  server.getPlayerProfileCache().getGameProfileForUsername(args[0]);
+
+            EntityPlayer player = null;
+
+            if (playerProfile != null)
+            {
+                player = (EntityPlayer) server.getEntityFromUuid(playerProfile.getId());
```

Comment:
You won't be able to get the player entity. Check in the other command where we use it to get the colony owner with the game profile.